packages/openapi-to-graphql/src/resolver_builder.ts

```diff
@@ -40,13 +49,272 @@ type GetResolverParams = {
+      if (
+        !(parentIdentifier.length === 0) &&
+        parentIdentifier in root['_openAPIToGraphQL'].data
+      ) {
+        /**
+         * Resolving link params may change the usedParams, but these changes
+         * should not be present in the parent _openAPIToGraphQL, therefore copy
+         * the object
+         */
+        resolveData = JSON.parse(
+          JSON.stringify(root['_openAPIToGraphQL'].data[parentIdentifier])
+        )
+      }
+    }
+
+    if (payloadName && typeof payloadName === 'string') {
+      // The option genericPayloadArgName will change the payload name to "requestBody"
+      const sanePayloadName = data.options.genericPayloadArgName
+        ? 'requestBody'
+        : Oas3Tools.sanitize(payloadName, Oas3Tools.CaseStyle.camelCase)
+
+      if (sanePayloadName in args) {
+        if (typeof args[sanePayloadName] === 'object') {
+          const rawPayload = Oas3Tools.desanitizeObjKeys(
+            args[sanePayloadName],
+            data.saneMap
+          )
+          resolveData.usedPayload = rawPayload
+        } else {
+          const rawPayload = JSON.parse(args[sanePayloadName])
+          resolveData.usedPayload = rawPayload
+        }
+      }
+    }
+
+    pubsubLog(
+      `Subscription schema : ${JSON.stringify(resolveData.usedPayload)}`
+    )
+
+    if (typeof resolveData.usedParams === 'undefined') {
+      resolveData.usedParams = {}
+    }
+
+    if (connectOptions) {
+      resolveData.usedRequestOptions = connectOptions
+    } else {
+      resolveData.usedRequestOptions = {
+        method: resolveData.usedPayload.method
+          ? resolveData.usedPayload.method
+          : method.toUpperCase()
+      }
+    }
+
+    let value = path
+    let paramNameWithoutLocation = paramName
+    if (paramName.indexOf('.') !== -1) {
+      paramNameWithoutLocation = paramName.split('.')[1]
+    }
+
+    // /**
+    //  * see if the callback path contains constants expression
+    //  *
+    //  */
+    if (value.search(/{|}/) === -1) {
+      args[paramNameWithoutLocation] = isRuntimeExpression(value)
+        ? resolveLinkParameter(paramName, value, resolveData, root, args)
+        : value
+    } else {
+      // Replace callback expression with appropriate values
+      const cbParams = value.match(/{([^}]*)}/g)
+      pubsubLog(`Analyzing subscription path : ${cbParams.toString()}`)
+
+      cbParams.forEach(cbParam => {
+        value = value.replace(
+          cbParam,
+          resolveLinkParameter(
+            paramName,
+            cbParam.substring(1, cbParam.length - 1),
+            resolveData,
+            root,
+            args
+          )
+        )
+      })
+      args[paramNameWithoutLocation] = value
+    }
+    const topic = args[paramNameWithoutLocation] || 'test'
+    pubsubLog(`Subscring to : ${topic}`)
+    return ctx.pubsub
+      ? ctx.pubsub.asyncIterator(topic)
+      : pubsub.asyncIterator(topic)
+  }
```

Comment:
If pubsub is found in context provided by the user, it can replace the one declared in this file.
Which is recommended to do [here](https://github.com/apollographql/graphql-subscriptions#getting-started-with-your-first-subscription)